src/frontend/src/views/Data/components/TreesView/components/BlankStateTrees/style.ts

```diff
@@ -0,0 +1,77 @@
+import styled from "@emotion/styled";
```

Comment:
Most of these styles are exact copies of the BlankState for Samples, but added here to avoid reaching across components.